v4/export/config.go

```diff
@@ -9,12 +10,19 @@ import (
 	"time"
 
 	"github.com/coreos/go-semver/semver"
-	"github.com/pingcap/dumpling/v4/log"
+	"github.com/pingcap/br/pkg/storage"
+	"github.com/pingcap/failpoint"
+	"github.com/pingcap/kvproto/pkg/backup"
 	filter "github.com/pingcap/tidb-tools/pkg/table-filter"
 	"go.uber.org/zap"
+
+	"github.com/pingcap/dumpling/v4/log"
 )
 
 type Config struct {
+	// TODO support passing in options via flags as well as toml
+	storage.BackendOptions
```

Comment:
supporting it should be as simple as using [`storage.DefineFlags`](https://pkg.go.dev/github.com/pingcap/br/pkg/storage#DefineFlags) and using [`.ParseFromFlags()`](https://pkg.go.dev/github.com/pingcap/br/pkg/storage#BackendOptions.ParseFromFlags) after constructing the `BackendOptions`.

Reply:
it was indeed simple thanks!